### ISSUE TITLE
Vim mode: Always in Insert mode, can't switch to Normal mode.

### ISSUE THREAD

**Issue by JikkuJose** [dbcli/pgcli#1063]
## Description
I am unable to switch to `Normal Mode` while I have selected the Vim mode in pgcli. Its always in Insert. Any ideas? (I pressed Escape a thousand times.)

This is what I see at the bottom of the window:

`[F2] Smart Completion: ON  [F3] Multiline: ON   (Semi-colon [;] will end the line) [F4] Vi-mode (I)`

Note: 
1. I am able to toggle the given settings using function keys (`Smart Completion`/`Multiline`/`Vi-mode`)
2. Neovim is my primary editor & I use it inside iTerm (so escape key works).
3. I have `tmux` installed
4. I have Vim mode in my CLI already

## Your environment
- [x] Mac 10.14.5
- [x] iTerm 2.1.4
- [x] `pip freeze`:

```
click==6.7
Flask==1.0.2
futures==3.0.3
gitdb2==2.0.0
GitPython==2.1.1
greenlet==0.4.9
itsdangerous==0.24
Jinja2==2.10
MarkupSafe==1.0
msgpack==0.6.1
msgpack-python==0.4.7
neovim==0.3.1
pynvim==0.3.2
six==1.10.0
smmap2==2.0.1
trollius==2.1
Werkzeug==0.14.1
```

**Comment by oblitum** [comment]
What's your  pgcli version? It sounds like a duplicate of #1038.

**Comment by j-bennet** [comment]
In #1038 I advised to install pgcli from master, but the fix was released in 2.1.1. Try updating from pip:

`pip install -U pgcli`

**Comment by JikkuJose** [comment]
@j-bennet & @oblitum thanks! That fixed it!
